src/main/drivers/max7456.c

```diff
@@ -455,11 +455,12 @@ void max7456Init(const vcdProfile_t *pVcdProfile)
         break;
     }
 
-    // XXX Disable for production
+#ifdef DEBUG_MAX7456_SPI_CLOCK
```

Comment:
Can we have this as a `debug_mode` please?

Reply:
okie dokie :)